backends/events/sdl/sdl-events.cpp

```diff
@@ -1077,3 +1091,434 @@ uint32 SdlEventSource::obtainUnicode(const SDL_Keysym keySym) {
 }
 
 #endif
+
+#if defined(USE_SDL_TS_VMOUSE)
+void SdlEventSource::_touchInitialize()
+{
+    // TODO - Start with a temporary window size before getting true values
+    _touchWindowWidth = 256;
+    _touchWindowHeight = 256;
+    _touchPanelWidth = 256;
+    _touchPanelHeight = 256;
+
+    // TODO - Temporary default values - Add to configuration?
+    _touchMaxTapTime = 250;
```

Comment:
Yes, it would be proper to put all of these to `ConfMan.registerDefault()`, and also put these to GUI